README.md

```diff
@@ -63,24 +63,24 @@ You must either construct an `o-tooltip` object or fire an `o.DOMContentLoaded`
 #### Constructing an o-tooltip
 If you have setup your tooltip declaratively, the following applies:
 ```js
-const oTooltip = require('o-tooltip');
-let tooltipEl = Document.getElementById('my-tooltip-element');
-const oTooltip = new oTooltip(tooltipEl, '#declarative-tooltip-target');
+const Tooltip = require('o-tooltip');
+const tooltipEl = Document.getElementById('my-tooltip-element');
+const tooltip = new Tooltip(tooltipEl);
 ```
 
 Alternatively, if you want to construct a tooltip imperatively, you can instantiate o-tooltip by passing in your target element and an options object.
 
 ```js
-const oTooltip = require('o-tooltip');
-let tooltipElement = document.querySelector('.imperative-tooltip-element');
+const Tooltip = require('o-tooltip');
+const tooltipEl = document.querySelector('.imperative-tooltip-element');
 const opts = {
 	target: 'tooltip-target-imperative',
 	content: 'Click to save to somewhere',
 	showOnConstruction: true,
 	position: 'right'
 }
 
-const oTooltip = new Tooltip(tooltipElement, opts);
+const tooltip = new Tooltip(tooltipEl, opts);
 ```
 
 Since this creates the tooltip from scratch, it is important to include any declarative attributes (as listed above) in the options object, in addition to:
```

Comment:
I don't understand the note about the `content` attribute below... it's certainly not required. I mean, the first argument provided to the constructor _is_ the content, isn't it?